src/game.cpp

```diff
@@ -3650,6 +3651,87 @@ bool game::save_player_data()
     }
 }
 
+void game::dump_stats( const std::string& what )
+{
+    load_core_data();
+    DynamicDataLoader::get_instance().finalize_loaded_data();
+
+    if( what == "GUN" ) {
+        std::cout
```

Comment:
`std::cout`? Sounds rather inflexible and unusual. Most of the on-load kind of data (materials, groups etc.) are dumped to the debug log.